recipe.cake

```diff
@@ -22,10 +22,22 @@ else
     Environment.SetVariableNames();
 }
 
+Func<FilePathCollection> getScriptsToVerify = () =>
+{
+    var scriptsToVerify = GetFiles(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/**/*.{ps1|psm1}");

```

Comment:
Any reason that this doesn't include `.psd1`?  They are included in line 40 below
```suggestion
    var scriptsToVerify = GetFiles(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/**/*.{ps1|psm1|psd1}");
```